tb/core/tb_top_verilator.cpp

```diff
@@ -84,7 +84,7 @@ void dump_memory()
 
         std::cout << "finished dumping memory" << std::endl;
 
-    } catch (std::ofstream::failure e) {
+    } catch (std::ofstream::failure &e) {
```

Comment:
As as comment above.

Reply:
Also GCC issues a warning if you use an instance not a reference.